manheim_cloudmapper/ses/report.py

```diff
@@ -0,0 +1,179 @@
+import os
+import datetime
+import logging
+from premailer import transform
+from .ses import SES
+
+logger = logging.getLogger(__name__)
+
+
+class Report():
```

Comment:
It appears to me that this class' only function is to send reports via SES. Assuming that's the case, please rename it accordingly to something like "EmailReport" or "SesReportSender".

Reply:
I can see your point here, but I also am doing some javascript and css changes to the cloudmapper default report in order to make them email friendly.

Reply:
Is that code ever executed if we're not sending email? i.e., looking at [send_email.py](https://github.com/manheim/manheim-cloudmapper/blob/4a397e18617d7a5a60a997e787c6d5daa6124181/manheim_cloudmapper/send_email.py):

```python
def send_email():
    if bool_convert(os.environ['SES_ENABLED']):
        Report().generate_and_send_email()
```

It looks like this class is never run unless we're sending email. Essentially I'm suggesting that we either (1) make the report class able to generate the HTML without sending it via email, or (2) if it's only ever going to send email too, be named accordingly.

Reply:
Yeah this code will get skipped entirely when ses is disabled. I will change the class name to 'SesReportSender'